swagger/v1/person_escort_record.yaml

```diff
@@ -100,6 +100,7 @@ PersonEscortRecord:
     relationships:
       type: object
       required:
+        - move
         - profile
```

Comment:
Do you want to remove profile as a required relationship reference, here?

Reply:
We do require both a move, and a move's profile to be always present relationships

Reply:
This still feels slightly confusing to me - the profile relationship is indirectly through the move, so I don't think it makes sense to specify it here as a required relationship of the PER, particularly as the PER schema makes no mention of the profile.

Reply:
Those values depend on the context of what we see those required relationships as, A PER cannot exist without both a move and its associated profile. If one is missing the PER creation fails. They both exist as reference ids on the table itself. We cannot remove the `profile_id` column as this supports legacy PERs. However if we see these relationships as what we pass in on creation then I can remove `profile` as well as `framework_responses` from `required.

Reply:
Sounds like we might want to migrate all historical PERs to not use the profile id?

Reply:
And then we can just do away with the profile as a required relationship in this document?

Reply:
A profile can be linked to multiple moves, meaning we can't determine the specific move related to the person escort record, the main reason we now require a Move instead of a Profile to create the PER. I would like to keep profile now, as we require a unique validation on Profile when creating the PER. But In reality we never attached a profile to multiple moves historically, so I don't think migrating might be problematic, but I'd like to keep it for now as later a profile can be linked to multiple moves when wanting to reuse the PER, so after clients implement that change I'm going to revisit the migration.

Reply:
Keep in mind that suppliers also create moves and may re-use a single profile across multiple moves -  especially for the same custody period. E.g. move1 to court; court hearing; move2 remand from court. So probably both move and profile are necessary here?

Reply:
Yea I think so, best keep them as is. I've also chatted with the FE and we're happy to keep both relationships for now as well, but only create a PER from a move 👍